app/models/service_template_transformation_plan_task.rb

```diff
@@ -132,20 +136,21 @@ def transformation_log
 
   # Intend to be called by UI to display transformation log. The log is stored in MiqTask#task_results
   # Since the task_results may contain a large block of data, it is desired to remove the task upon receiving the data
-  def transformation_log_queue(userid = nil)
+  def transformation_log_queue(userid = nil, log_type = 'v2v')
+    raise "Transformation log type '#{log_type}' not supported" unless valid_transformation_log_types.include?(log_type)
     userid ||= User.current_userid || 'system'
     if conversion_host.nil?
-      msg = "Conversion host was not found. Cannot queue the download of transformation log."
+      msg = "Conversion host was not found. Cannot queue the download of #{log_type} log."
       return create_error_status_task(userid, msg).id
     end
 
-    _log.info("Queuing the download of transformation log for #{description} with ID [#{id}]")
+    _log.info("Queuing the download of #{log_type} log for #{description} with ID [#{id}]")
     task_options = {:userid => userid, :action => 'transformation_log'}
     queue_options = {:class_name  => self.class,
                      :method_name => 'transformation_log',
                      :instance_id => id,
                      :priority    => MiqQueue::HIGH_PRIORITY,
```

Comment:
Not related to this change but...why is this HIGH_PRIORITY??  @fdupont-redhat can you put in a follow-up PR to remove the priority property? (NORMAL is the default so we dont' need to pass anything in)

Reply:
Created the PR: https://github.com/ManageIQ/manageiq/pull/18524